packages/manager/modules/pci/src/projects/project/private-networks/private-networks.less

```diff
@@ -0,0 +1,18 @@
+.pv-data-grid {
+  .oui-datagrid-row-detail__container {
+    padding-top: 0;
+  }
+  .grid-details-table {
+    th,
+    tr {
+      &:first-child {
+        min-width: 4rem;
+        width: 4rem;
+      }
```

Comment:
**question:** Instead of adding an empty column, why not add padding to the first column?